fpna/Utils/ExtentionMethods.ipynb

```diff
@@ -108,9 +133,9 @@
         {
             "cell_type": "code",
             "source": [
-                "public static double[] GetBoundedRandomCashflow(this int shift, int length, double max, double min){",
+                "public static double[] GetBoundedRandomValues(this int shift, int length, double max, double min){",
```

Comment:
I think this method should include the InitialValue. 
I think the BoundedCashflow has that property

Reply:
- I think this should give purely random number. 
- we will add later some statistical distributions where the initialValue will be mean. but for now I prefer to keep it now.